docs/deploy-guide/gitops-install.md

```diff
@@ -193,8 +188,7 @@ kubectl label node $(kubectl get nodes -o 'jsonpath={.items[*].metadata.name}')
 Finally run the following to have ArgoCD deploy the system:
 
 ```bash
-# from your understack checkout
-kubectl apply -f apps/aio-app-of-apps.yaml
+kubectl -n argocd apply -f https://raw.githubusercontent.com/rackerlabs/understack/main/apps/aio-app-of-apps.yaml
```

Comment:
I am conflicted about this one. It's good that it does not require user to pop back into understack's directory, but it's not so good because we cannott offer any guarantees that version at `main` branch is compatible with the version that user has checked out locally.

Do we have to change to the $UC_DEPLOY directory at all now that you solved the `git -C` thing?

Reply:
That's a good point. I can update this as well.